types.go

```diff
@@ -0,0 +1,120 @@
+package fq
+
+import (
+	"fmt"
+	"strings"
+)
+
+// TODO(aaron-prindle) currently testing with one concurrent request
+const C = 1 // const C = 300
+
+// TODO(aaron-prindle) currently service time "G" is not implemented entirely
+const G = 100 // const G = 60000
```

Comment:
This needs to be in the same units as virtual time.

Reply:
This was a (bad) estimate.  The units here are milliseconds.  G = 6000ms is the 1 minutes time from the doc.  100 was a smaller value that I was playing with.  In the tests requests are processed  ~instantly.